extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/builder/BuildServerBuilder.java

```diff
@@ -55,8 +58,42 @@ protected IProject[] build(int kind, Map<String, String> args, IProgressMonitor
                     throw new CoreException(new Status(IStatus.ERROR, ImporterPlugin.PLUGIN_ID,
                             IResourceStatus.BUILD_FAILED, "Build Failed.", null));
                 }
+                this.refreshOutputs(monitor);
             }
         }
         return null;
     }
+
+    /**
+     * Trigger <code>.refreshLocal()</code> to all the output folders of the project.
+     * This is to make sure the changes made by the build server are reflected in the workspace.
+     */
+    private void refreshOutputs(IProgressMonitor monitor) throws CoreException {
+        IJavaProject javaProject = ProjectUtils.getJavaProject(this.getProject());
+        if (javaProject == null) {
+            return;
+        }
+
+        boolean needRefreshDefaultOutput = false;
+        for (IClasspathEntry cp : javaProject.getRawClasspath()) {
+            if (cp.getEntryKind() != IClasspathEntry.CPE_SOURCE) {
+                continue;
+            }
+
+            IPath output = cp.getOutputLocation().removeFirstSegments(1);
```

Comment:
what does it mean by removeFirstSegments?

Reply:
the first segment is the project name, which needs to be removed to get the relative path